include/OsmAndCore/Data/ObfPoiSectionInfo.h

```diff
@@ -98,6 +99,8 @@ namespace OsmAnd
         uint32_t firstBoxInnerOffset;
         mutable QHash<uint32_t, QList<QPair<QString, QString>>> tagGroups;
         mutable BBoxIndexTree bboxIndexCache;
```

Comment:
looks wrong, we shouldn't introduce any locks / concurrency to readers. Readers are solely 1 thread and nobody should ever call reading API concurrently as it causes multiple bugs